suseconnect-ng.spec

```diff
@@ -31,7 +31,13 @@ Group:          System/Management
 Source:         connect-ng-%{version}.tar.xz
 Source1:        %name-rpmlintrc
 BuildRequires:  golang-packaging
+# use FIPS compliant go version for SLE targets
+%if 0%{?sle_version}
```

Comment:
We need to exclude Leap 15.3 and 15.4 here. 15.5 seems to build.

Reply:
so it seems `sle_version` is also set on Leap. Should this conditional also check that `is_opensuse` is not set?
https://en.opensuse.org/openSUSE:Packaging_for_Leap#RPM_Distro_Version_Macros

Reply:
the condition should be (I don't know the correct syntax from memory): sle_version is set or ( is_opensuse = 1 and sle_version >= 150500 )